src/history/HistoryTests.cpp

```diff
@@ -609,11 +610,14 @@ TEST_CASE("Catchup manual", "[history][catchupmanual]")
                 auto a = catchupSimulation.catchupNewApplication(
                     configuration.toLedger(), configuration.count(), true,
                     dbMode, name);
+                REQUIRE(a);
                 // manual catchup-complete to first checkpoint
-                catchupSimulation.catchupApplication(
-                    initLedger1, std::numeric_limits<uint32_t>::max(), true, a);
+                REQUIRE(catchupSimulation.catchupApplication(
+                    initLedger1, std::numeric_limits<uint32_t>::max(), true,
+                    a));
```

Comment:
I think 'a' variable is not longer necessary here. You can just do REQUIRE on line 610 and use plain 'true' on 617 and 620.

Reply:
ok, for some reason I was expecting this to be bool.